src/utils/predicates.ts

```diff
@@ -11,3 +11,29 @@ export function isNotNullish<T extends NonNullable<unknown>>(
 ): value is T {
 	return value !== null && value !== undefined;
 }
+
+export function isString(value: unknown): value is string {
+	return typeof value === "string";
+}
```

Comment:
[Refactor] I don't think we need such a straightforward function. Just a `typeof` in-place is code is fine.

```suggestion
```